app-api/src/main/java/org/openmrs/eip/app/CustomFileOffsetBackingStore.java

```diff
@@ -11,76 +11,41 @@
  */
 public class CustomFileOffsetBackingStore extends FileOffsetBackingStore {
 	
```

Comment:
I propose you instead merge your new backing store class into this and DO NOT alter the logic in this

Reply:
Done.